src/filter.js

```diff
@@ -81,6 +81,13 @@ cartodb.d3.extend(Filter.prototype, cartodb.d3.Event, {
     return this.dimensions['tiles'].top(Infinity)
   },
 
+  getColumnValues: function (column, numberOfValues) {
+    if (!this.dimensions[column]) {
```

Comment:
We could use `this.createDimension(column)` here. `createDimension` seems to be private so I'd prefix it with an `_`.